src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
```

Comment:
The variable name 'smallDiscount' is not clear enough. Rename it to 'smallDiscountAmount' to indicate it's the discount amount for rentals of 3 days or more. 🌟